include/macros.h

```diff
@@ -85,6 +87,11 @@
         View_SetViewport(view, &viewport); \
     }
 
+#define	CHECK_IPAGE(p) (((p).ipage >= pfs->inodeStartPage) && \
+				 ((p).inode_t.bank < pfs->banks) && \
+				 ((p).inode_t.page >= 0x01) && \
+				 ((p).inode_t.page < 0x80))
```

Comment:
And I'm psure this one would belong in pfs.h or whatever it's called.

Reply:
See above comment on CHECK_IPAGE